app/code/GorbanSv/CustomerRegistrationPopup/view/frontend/requirejs-config.js

```diff
@@ -0,0 +1,8 @@
+const config = {
+    paths: {
+        dealerRegistration: 'GorbanSv_CustomerRegistrationPopup/js/popup',
+    },
+    shim: {
```

Comment:
No need to create a shim, because you use `require` in your JS code to import dependencies